yearn/contracts/ConvexStrategy.sol

```diff
@@ -606,7 +613,23 @@ contract ConvexStrategy is BaseStrategy {
         override
         returns (uint256)
     {
-        // TODO: Create an accurate price oracle.
-        return amtInWei;
+        address[] memory path = new address[](2);
+        path[0] = weth;
+        path[1] = tbtc;
+
+        // LP token (Curve.fi tBTC/sbtcCrv) consists of 4 tokens: tBTC, renBTC,
+        // wBTC and sBTC. As of writing this contract, there's no pool available
+        // that trades 'Curve.fi tBTC/sbtcCrv' with ETH. However, all of these
+        // 4 tokens are pegged to bitcoin and hold similar value. In order to estimate
+        // profitability of calling 'harvest()' we need to workaround and see the
+        // rate between tBTC - ETH on dex like Uniswap. tBTC was chosen arbitrary
+        // since it is availbale on Uniswap and was built on the keep-network.
+        // amounts[0] -> ETH in wei
+        // amounts[1] -> tBTC
+        uint256[] memory amounts = IUniswapV2Router(uniswap).getAmountsOut(amtInWei, path);
+
+        // By knowing ETH->tBTC amounts it is possible to estimate the amount of
+        // 'Curve.fi tBTC/sbtcCrv' LP tokens to be received for tBTC tokens.
+        return ICurvePool(tbtcCurvePoolDepositor).calc_token_amount([amounts[1], 0, 0, 0], false);
```

Comment:
The logic seems correct but I'd change those comments a little bit. They are tightly tied to the tBTC v1 and its Curve pool internals. My proposition:
```
// Convert the ETH amount denominated in WEI into an amount denominated in one of the tokens accepted by 
// the tBTC v2 Curve pool using Uniswap. The tBTC v2 token was chosen arbitrarily since it is available on Uniswap
// and was built on the Keep Network.
uint256[] memory amounts = IUniswapV2Router(uniswap).getAmountsOut(amtInWei, path);

// Use the amount denominated in tBTC v2 to calculate the amount of LP token (vault's underlying token) that 
// could be obtained if that tBTC v2 amount was deposited in the tBTC v2 Curve pool. This way we obtain an
// estimated value of the original WEI amount represented in the vault's underlying token
return ICurvePool(tbtcCurvePoolDepositor).calc_token_amount([amounts[1], 0, 0, 0], false);
```
WDYT?

Reply:
Sure, good points, I've realised that after your first comment regarding changing the tile of this PR. I've changed the original comment as you suggested with just some minor changes e247269